Example in README doesn't work

Attempting to compile the "Simple example" section of the README gives the following errors:

```
error[E0599]: no method named `write_str` found for type `tabwriter::TabWriter<std::vec::Vec<u8>>` in the current scope
  --> src/main.rs:10:4
   |
10 | tw.write_str("
   |    ^^^^^^^^^

error[E0599]: no method named `as_slice` found for type `std::string::String` in the current scope
  --> src/main.rs:20:20
   |
20 | assert_eq!(written.as_slice(), "
   |                    ^^^^^^^^
```

--- a/README.md
+++ b/README.md
@@ -12,10 +12,12 @@ Dual-licensed under MIT or the [UNLICENSE](http://unlicense.org).
 ### Simple example of library
 
 ```rust
+use std::io::Write;
+
 use tabwriter::TabWriter;
 
 let mut tw = TabWriter::new(vec![]);
-tw.write_str("
+tw.write_all(b"
 Bruce Springsteen\tBorn to Run
 Bob Seger\tNight Moves
 Metallica\tBlack
@@ -25,7 +27,7 @@ tw.flush().unwrap();
 
 let written = String::from_utf8(tw.into_inner().unwrap()).unwrap();
 
-assert_eq!(written.as_slice(), "
+assert_eq!(&written, "
 Bruce Springsteen  Born to Run
 Bob Seger          Night Moves
 Metallica          Black

